### PR TITLE
add(library): STDF

### PR DESCRIPTION
**Pull request by dufu1991**
Add STDF - Mobile web component library based on Svelte and Tailwind.

| Name                 | Link                 |
| -------------------- | -------------------- |
| STDF | https://stdf.design |

_Resource description here_

---

- [x] My item is in the right category
- [x] My item is logically grouped below similar items
- [x] My item's name and description respects the conventions of the list
- [x] My item is awesome
- [x] My item is in line with the [Tailwind brand usage guidelines](https://tailwindcss.com/brand#usage)
- [x] I have read and followed the [contribution guidelines](https://github.com/aniftyco/awesome-tailwindcss/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

**Comment by innocenzi** [comment]
Hey, I feel like this isn't really about Tailwind, the library just happens to use it. What do you think @joshmanders?

**Review by joshmanders — DISMISSED** [review]
Hmmm it's not about tailwind but it is definitely a tailwind thing, and there's so many react/vue stuff and like 1 svelte, maybe we can allow it to give more diversity to other frameworks?

I think it looks good and is pretty complete, not someone spending a day to whip up a few components and now wants to garner some github stars as easily as possible.

I approve, I'll let you make the final decision.

File: README.md [hunk 1]
```diff
@@ -277,6 +277,7 @@
 - 📚 [Built At Lightspeed](https://www.builtatlightspeed.com/) - Massive directory of 500+ Tailwind templates, starters and UI kits.
 - 📚 [Statichunt](https://statichunt.com/tailwind-templates) - Open source directory of hand-picked free and premium Tailwind templates & Starters.
 - 📚 [Tailkits](https://tailkits.com/) - Curated Tailwind CSS components, templates, UI kits, resources, tools & more.
+- 📚 [STDF](https://stdf.design) - Mobile web component library based on Svelte and Tailwind.
```
Comment (innocenzi) [inline README.md:280]:
```suggestion
- 📚 [STDF](https://stdf.design) - Mobile web component library based on Svelte and Tailwind CSS.
```

**Review by innocenzi — APPROVED** [review]

**Comment by innocenzi** [comment]
Fine by me 👌

**Pull request merged**